dbt/scripts/check_sort_dbt_yaml_files.py

```diff
@@ -61,6 +61,11 @@ def check_columns(file_path):
     try:
         with open(file_path, "r") as file:
             first_line = file.readline().strip()
+            # This disregards files within the check_columns() function
+            # that have this tag at the top. This is limited to yaml
+            # files with column sorting, if we want to expand we will
+            # have to generalize it or manually add a check to the
+            # other functions
```

Comment:
**[Nitpick, blocking]** Hmm, I don't think this solves the `file.seek(0)` issue I was thinking about in https://github.com/ccao-data/data-architecture/pull/578/files/8fffe4512174dfb754ef13d9e01d3f8acedc69b7..e7c6802ce0c85725645320a644be6be41a44f8c0#r1746019955. Maybe we can discuss on Monday?